src/cli/backup/backup.go

```diff
@@ -225,6 +226,12 @@ func runBackups(
 
 		err = bo.Run(ictx)
 		if err != nil {
+			if errors.Is(err, graph.ErrServiceNotEnabled) {
+				logger.Ctx(ctx).Infof("Service not enabled for creating backup for %s", bo.ResourceOwner.Name())
```

Comment:
Per logging standards, the message should avoid capitals and formatting.
```suggestion
				logger.Ctx(ctx).Infow("service not enabled", "resource_owner_name", bo.ResourceOwner.Name())
```